### ISSUE TITLE
Mine blocks [ Iron, emerald, diamond etc.. ] 

### ISSUE THREAD

**Issue by Libertyxo** [michaelzangl/minebot#242]
Is it possible to make it target ore blocks too not just mine ores?

**Comment by nightscloud2** [comment]
Yes, there are several ways, although i kinda forget. Been a while since i've done it. There is a way to change the priority so that it prioritizes the ore you want. By default, rare ore has higher priority, so if you're equal distance from iron and diamond, it should go for the diamonds. You can modify that priority and make iron higher than diamonds. Or, you can use the javascript api minescript.doStrategy(minescript.strategy("minebot", "mine", "iron_ore")); for example which would mine iron ore, however i think in the latest updates you may need to use "minecraft:iron_ore".
